backend/codesirius/authentication/models/verification.py

```
@@ -4,6 +4,7 @@
 from django.contrib.auth import get_user_model
 from django.db import models
 from django.utils.timezone import now, timedelta
+from rest_framework.exceptions import APIException, ValidationError
```

Ruff (0.8.2)

`rest_framework.exceptions.APIException` imported but unused

Remove unused import: `rest_framework.exceptions.APIException`

(F401)